src/server/stat_job.c

```diff
@@ -302,6 +303,17 @@ status_job(job *pjob, struct batch_request *preq, svrattrl *pal, pbs_list_head *
 	append_link(pstathd, &pstat->brp_stlink, pstat);
 	preq->rq_reply.brp_count++;
 
+	/* Temporarily set suspend/user suspend states for the stat */
+	if (check_job_state(pjob, JOB_STATE_LTR_RUNNING)) {
```

Comment:
Looks like this is newly added, can please add comments why we need to temporarily set different state? And this is status_job() means it will be visible on client side right? if true then don't we need some kind a design so that we can update new stat in docs/man pages?

Reply:
No external change, behavior to user is the same. Earlier server code was storing numeric state values in the quick save area and letter state values in the attribute list. So, states U and S were set inside the attribute list and were reported only for stats. Now that we've removed the quick save area, there's no way to store numeric values anymore, but the code still doesn't expect U and S to be part of the state value in the server. So, we set it temporarily here only for stat, the rest of the server can work like before.